modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/Validate.java

```diff
@@ -44,12 +46,20 @@ public class Validate extends OpenApiGeneratorCommand {
     @Option(name = { "--recommend"}, title = "recommend spec improvements")
     private Boolean recommend;
 
+    @Option(
+            name = {"-a", "--auth"},
+            title = "authorization",
+            description = "adds authorization headers when fetching the OpenAPI definitions remotely. "
+                    + "Pass in a URL-encoded string of name:header with a comma separating multiple values")
+    private String auth;
+
```

Comment:
thanks for the PR

looks like this is the same as `--auth` option for generate 

https://github.com/OpenAPITools/openapi-generator/blob/45e1f73f7aec894e3d2765a90729ba2b6f82dee9/modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/Generate.java#L76-L81